cmd/crates/soroban-spec-tools/src/lib.rs

```diff
@@ -121,6 +121,9 @@ impl Spec {
             | ScType::I256
             | ScType::String
             | ScType::Bool => String::new(),
+            ScType::MuxedAddress => {
+                String::from("Can be public key (G13..) or a muxed account (M13..)")
```

Comment:
Sorry, actually it can be even more things:
```suggestion
                String::from("Can be public key (G13..), contract ID (C13...), or a muxed account (M13..), or an identity")
```